test/core/polling_place/TestPollingPlace.sol

```diff
@@ -1,499 +0,0 @@
-pragma solidity ^0.4.24;
-
-// Copyright 2018 OpenST Ltd.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import "truffle/Assert.sol";
-import "truffle/DeployedAddresses.sol";
-import "../../../contracts/test/test_lib/RevertProxy.sol";
-import "../../../contracts/test/core/BlockStoreMock.sol";
-import "../../../contracts/core/PollingPlace.sol";
-
-/**
- * @title Wrapper to wrap polling place methods.
- *
- * @notice The wrapper is required to drop the return value from the method
- *         under test. See also:
- *         https://github.com/trufflesuite/truffle/issues/1001#issuecomment-39748268
- *
- * @dev Note that the wrapped methods do not have a return value. It is
- *      required to drop it in order for the low-level call of the RevertProxy
- *      to work.
- */
-contract PollingPlaceWrapper {
-
-    /** The wrapped contract. */
-    PollingPlace pollingPlace;
-
-    /**
-     * @notice Setting the wrapped construct. It is not necessarily known at
-     *         construction.
-     *
-     * @param _pollingPlace The address of the wrapped contract.
-     */
-    function setPollingPlace(address _pollingPlace) public {
-        pollingPlace = PollingPlace(_pollingPlace);
-    }
-
-    /**
-     * @notice Wrapper function for the wrapped `updateMetaBlockHeight`,
-     *         without a return value.
-     *
-     * @param _auxiliaryAddresses The addresses of the validators.
-     * @param _weights The weights of the validators.
-     * @param _originHeight The height of the origin chane where the new
-     *                      meta-block opens.
-     * @param _auxiliaryHeight The height of the auxiliary checkpoint that is
-     *                         the last finalised checkpoint within the
-     *                         previous, closed meta-block.
-     */
-    function updateMetaBlockHeight(
-        address[] _auxiliaryAddresses,
-        uint256[] _weights,
-        uint256 _originHeight,
-        uint256 _auxiliaryHeight
-    )
-        public
-    {
-        pollingPlace.updateMetaBlockHeight(
-            _auxiliaryAddresses,
-            _weights,
-            _originHeight,
-            _auxiliaryHeight
-        );
-    }
-}
-
-/**
- * @title Test contract to test the external methods of PollingPlace.
- */
-contract TestPollingPlace {
```

Comment:
Why was this file deleted? 😱

Reply:
This was test related to `updateMetaBlockHeight` in solidity.
I have created `update_meta_block_height.js` that covers all the tests there.

Reply:
🤷‍♂️

Reply:
Ahh, let me explain more.  This file needed changes related to `PollingPlaceWrapper`. As we decided in Pune that we will write the tests in js. So it was easy for me to move it to js. All the tests are already covered in the new file.

Let me know if you don't agree with the change. I will revert and make the changes in the sol file itself.